localwallet/localwallet.go

```diff
@@ -0,0 +1,67 @@
+package localwallet
```

Comment:
This is a local wallet to store the provided private keys, and provide an interface implementation for the [Wallet interface](https://github.com/textileio/go-auctions-client/pull/1/files#diff-6dc4054cf1607c7c1e1415ba59f68670659a308d79e333c71429d86d7d8b0b2dR36). Basically contains a map of keys and uses `go-filsigner` to do the heavy lifting.

People running our code as a lib can provide other implementations if keys live somewhere else.